docs/en/week11/11-2.md

```diff
@@ -0,0 +1,62 @@
+---
+lang-ref: ch.11-2
+lecturer: Awni Hannun
+title: Speech Recognition and Graph Transformer Network (Part 2)
+authors: Gyanesh Gupta, Abed Qaddoumi
+date: 14th April 2021
+---
+
+## Inference Time
+
+The inference of a transcription from a given audio signal can be formulated using 2 distributions:
+
+* The acoustic model (audio to transcription), represented as $P(Y|X)$
+* The language model, $P(Y)$
+
+The final inference is obtained by taking sum of the log probabilities of the above two, i.e.
+
+$$ Y* = \underset{Y}{\operatorname{argmax}} \log P(Y|X) + \log P(Y) $$
+
+We use the additional term to ensure the transcription is consistent with the rules of the language. Else we may get grammatically wrong transcription.
+
+
+## Beam Search
```

Comment:
Everything is still broken. Plz fix.

![image](https://user-images.githubusercontent.com/2119355/118412087-0e6ee480-b666-11eb-830d-4de390e7e3d9.png)

Reply:
Plz, read the [instructions](https://atcold.github.io/pytorch-Deep-Learning/en/about/).